action/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
+
+RUN pip install --no-cache-dir ptr
```

Comment:
Should we pre-install all the deps in https://github.com/facebookincubator/ptr/blob/master/requirements.txt so they are there in the image? Maybe even allow pip to create the cache-dir for these to help with new venv creations?

Reply:
Good idea, we should definitely install the other deps!